third_party/proton/tutorials/matmul.py

```diff
@@ -255,6 +236,12 @@ def grid(META):
     return c
 
 
+argparser = argparse.ArgumentParser()
+argparser.add_argument("--profile", action="store_true")
+argparser.add_argument("--cudagraph", action="store_true", default=False)
```

Comment:
Is it worth breaking this out into a separate "profiling graphs" tutorial and keeping this as generic matmul profiling example?